build.sbt

```diff
@@ -20,9 +20,7 @@ libraryDependencies += "org.apache.commons" % "commons-compress" % "1.4.1"
 
 libraryDependencies += "org.tukaani" % "xz" % "1.5"
  
-libraryDependencies += "org.apache.ant" % "ant" % "1.8.2"
-
-libraryDependencies += "org.testng" % "testng" % "6.8.8"
+libraryDependencies += "org.testng" % "testng" % "6.8.8" % "test"
```

Comment:
@cmnbroad  These changes look fine to me from an htsjdk perspective.  testng is only used in tests.  We could probably just remove it from the sbt completely since we don't actually use the sbt for anything other than publishing to maven.  

I can't see any usage of the ant dependency either.  It's possible it's needed for the ant build, but that would be unaffected by removing it.  Likewise, picard probably requires these be included by htsjdk, but since it doesn't access them through maven it won't matter if we remove them from the sbt.